server/src/main/java/io/littlehorse/server/KafkaStreamsServerImpl.java

```diff
@@ -71,105 +72,8 @@
 import io.littlehorse.common.util.LHProducer;
 import io.littlehorse.common.util.LHUtil;
 import io.littlehorse.sdk.common.exception.LHMisconfigurationException;
-import io.littlehorse.sdk.common.proto.ACLAction;
-import io.littlehorse.sdk.common.proto.ACLResource;
-import io.littlehorse.sdk.common.proto.AssignUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.AwaitWorkflowEventRequest;
-import io.littlehorse.sdk.common.proto.CancelUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.CompleteUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteExternalEventDefRequest;
-import io.littlehorse.sdk.common.proto.DeletePrincipalRequest;
-import io.littlehorse.sdk.common.proto.DeleteTaskDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteUserTaskDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteWfRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteWfSpecRequest;
-import io.littlehorse.sdk.common.proto.ExternalEvent;
-import io.littlehorse.sdk.common.proto.ExternalEventDef;
-import io.littlehorse.sdk.common.proto.ExternalEventDefId;
-import io.littlehorse.sdk.common.proto.ExternalEventDefIdList;
-import io.littlehorse.sdk.common.proto.ExternalEventId;
-import io.littlehorse.sdk.common.proto.ExternalEventIdList;
-import io.littlehorse.sdk.common.proto.ExternalEventList;
-import io.littlehorse.sdk.common.proto.GetLatestUserTaskDefRequest;
-import io.littlehorse.sdk.common.proto.GetLatestWfSpecRequest;
-import io.littlehorse.sdk.common.proto.LHHostInfo;
-import io.littlehorse.sdk.common.proto.ListExternalEventsRequest;
-import io.littlehorse.sdk.common.proto.ListNodeRunsRequest;
-import io.littlehorse.sdk.common.proto.ListTaskMetricsRequest;
-import io.littlehorse.sdk.common.proto.ListTaskMetricsResponse;
-import io.littlehorse.sdk.common.proto.ListTaskRunsRequest;
-import io.littlehorse.sdk.common.proto.ListUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.ListVariablesRequest;
-import io.littlehorse.sdk.common.proto.ListWfMetricsRequest;
-import io.littlehorse.sdk.common.proto.ListWfMetricsResponse;
+import io.littlehorse.sdk.common.proto.*;
```

Comment:
I don't think this is what we want...

Reply:
:porg: That is not what I wanted to do. Maybe `gradle spotlessApply` did that?

Reply:
![image](https://github.com/littlehorse-enterprises/littlehorse/assets/100447728/cb1489ae-e5ab-4d10-a873-37bbc82c63da)
That's what happened :joy: